proto/attest.proto

```diff
@@ -43,7 +43,7 @@ message Attestation {
   oneof tee_attestation {
     sevsnp.Attestation sev_snp_attestation = 8;
     tdx.QuoteV4 tdx_attestation = 9;
-  }
+  };
```

Comment:
Why is there a semicolon here?

Reply:
I was trying to add a new field in the message. Removed